core/src/avm2/globals/flash/media/Video.as

```diff
@@ -0,0 +1,19 @@
+package flash.media {
+    public final class Video {
```

Comment:
it's missing a DisplayObject inheritance; also it's not `final`

Reply:
Agree on `DisplayObject ` inheritance, but actually `Video` is not declared as `final` as per https://help.adobe.com/en_US/FlashPlatform/reference/actionscript/3/flash/media/Video.html.

Reply:
>  actually Video is not declared as final

yeah, that's what I meant :) That comment was from the version of this PR that had `final`.